actors/power/src/types.rs

```diff
@@ -65,3 +66,26 @@ pub struct CurrentTotalPowerReturn {
     pub pledge_collateral: TokenAmount,
     pub quality_adj_power_smoothed: FilterEstimate,
 }
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
```

Comment:
How do we want to serialize these? Should they be serialized as structs (as currently written), or their inner type (StoragePower / Address)?

Reply:
I would have said bare inner types, but above have suggested adding a second member to each of these.

Reply:
Discussed offline: We're going to maintain existing convention, which is for single-element structs to be serialized transparently.

Reply:
Agree with the result. I personally would skip the struct and just return StoragePower directly, but I'm not adamant. If we use the struct, we should go add them to all the other cases where we return a raw value.

Reply:
I do think I prefer to have an explicit struct. We're already inconsistent about whether we do this (eg. Market::WithdrawBalanceReturn, vs. Miner::change_owner_address).

I'll do a quick audit and change any cases I find, but I'd rather not be especially thorough.

Reply:
Actually, this should be a separate PR regardless. #817